### PR TITLE
Update zh_Hans translation

### PR DESCRIPTION
**Pull request by Artoria2e5**
This finishes all 200 messages and fixes some known translation errors, like mistaking "new" as an adjective in "new argument".

Honestly, I consider it a good idea to host all those translations on a translation platform like transifex.com or TranslationProject.org. Both of them are free-of-charge to open-source projects.

By the way, putting compiled MOs in the repo makes my commit fat..

### CODE REVIEW

File: web/locale/zh_Hans/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -249,9 +249,7 @@ msgstr "忘记密码了?"
 
 #: templates/auth/login_popup.html:6
 #, python-format
-msgid ""
-"<a href=\"%(login_url)s\">Login</a> or <a href=\"%(register_url)s"
-"\">register</a> to add a premise for this argument."
+msgid "<a href=\"%(login_url)s\">Login</a> or <a href=\"%(register_url)s\">register</a> to add a premise for this argument."
```
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: Oops, I should msgfmt these. Rebasing.

Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: msgcat *

File: web/locale/zh_Hans/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -758,7 +765,6 @@ msgstr "信息源："
 msgid "%(count)s fallacy reported."
 msgid_plural "%(count)s fallacies reported."
 msgstr[0] "%(count)s个谬误报告。"
-msgstr[1] "%(count)s个谬误报告。"
```
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: Applied correct plural for Chinese.

File: web/locale/zh_Hans/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
 msgstr ""
 "Project-Id-Version: Arguman\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2015-11-15 13:34+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: tvvocold <tvvocold@gmail.com>\n"
-"Language: zh-CN\n"
+"PO-Revision-Date: 2015-11-15 20:37-0500\n"
+"Last-Translator: Mingye Wang (Arthur2e5) <arthur200126@gmail.com>\n"
+"Language: zh_CN\n"
```
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: Uh, my editor thought I was writing for zh_CN, the *locale* name.

**Comment by Arthur2e5** [comment]
Hey, do you guys provide a POT file?

**Comment by Arthur2e5** [comment]
Wait, I should plug my name into the contributors list too.

**Comment by Arthur2e5** [comment]
```PO
#: templates/partials/notifications.html:19
msgid "You have no new notifications."
msgstr "你有新的消息。"
```

ARE YOU KIDDING?

File: web/locale/zh_Hans/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -280,22 +282,22 @@ msgstr "无效的密码重置链接。"
...
-"       我们已发送密码重置邮件。请查收您的 邮箱.\n"
+"       我们已发送密码重置邮件。请查收您的收件箱.\n"
 "    "
 
 #: templates/auth/password_reset_email.html:2
 msgid ""
 "You're receiving this email because you requested a password reset for your "
 "user account at arguman.org"
-msgstr ""
+msgstr "您还到这封电子邮件，是因为您要求重置您的 arguman.org 账户密码"
```
Comment (tvvocold) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: 还 > 收

File: web/locale/zh_Hans/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -320,7 +322,7 @@ msgstr "关注"
 
 #: templates/auth/profile.html:32 templates/stats/profile.html:21
 msgid "KARMA"
-msgstr ""
+msgstr "节操"
```
Comment (tvvocold) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: “节操” 在这里不合适，改成“人品值” 或其他词汇比较好点。

File: web/locale/zh_Hans/LC_MESSAGES/django.po (hunk 4 above)
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: Truly and indeed.

File: web/locale/zh_Hans/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -402,7 +404,7 @@ msgstr "登录"
 
 #: templates/base.html:41
 msgid "new argument"
-msgstr "新的论点"
+msgstr "新论点"
```
Comment (tvvocold) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: “新建”吧

**Comment by Arthur2e5** [comment]
I've had enough of rebase and force-pushing, but in order to make sure the repo is not filled with compiled MO bindiff garbage……

File: web/locale/zh_Hans/LC_MESSAGES/django.po (hunk 6 above)
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: Works in zh_CN UI.

File: web/locale/zh_Hans/LC_MESSAGES/django.po (hunk 5 above)
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: Waiting for my IR parser to dump a meaning for each of the words in the contexts I have learned so I can make a diff.

File: web/locale/zh_Hans/LC_MESSAGES/django.po (hunk 6 above)
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: But, well, yours is better.

File: web/locale/zh_Hans/LC_MESSAGES/django.po (hunk 5 above)
Comment (Arthur2e5) [inline web/locale/zh_Hans/LC_MESSAGES/django.po]: According to IR parser, both 人品(值) and 节操 give a informal feeling, just like KARMA in games like Fallout. Rejecting change.

**Pull request closed without merging**